wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -221,6 +221,48 @@
             "description": "Describes whether projects sources should be mount to the tool. `CHE_PROJECTS_ROOT` environment variable should contains a path where projects sources are mount",
             "default": "false"
           },
+          "command": {
+            "oneOf": [
+              {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              {
+                "type": "string"
```

Comment:
As far as I see `io.fabric8.kubernetes.api.model.Container` as List<String>` for `command` and `args`.
Also, I see that kubernetes docs contains examples with arrays (one or more elements) https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/.

I got the following exception when tried to parse container with 
```yaml
      args: 'echo'
      command: ['echo']
```
specified

![screenshot_20190227_154552](https://user-images.githubusercontent.com/5887312/53494662-ec99df00-3aa6-11e9-893a-ea0ee51b419a.png)

If no - maybe we should not too for simplification. WDYT?

Reply:
It's what OpenShift console says 
![screenshot_20190227_164023](https://user-images.githubusercontent.com/5887312/53498167-78633980-3aae-11e9-83e5-8576a3546e67.png)
when try to create a pod with containers with 
```
    - command: 'tail'
      args: ['-f', '/dev/null']
```
specified

Reply:
The `command` and `args` can only be arrays now.